lib/cli.js

```diff
@@ -24,6 +25,8 @@ updateNotifier({ pkg }).notify();
 
 if (argv.v) {
   console.log(`v${pkg.version}`);
+} else if (!isGit()) {
```

Comment:
It looks a bit weird if the else got multiple lines.

Actually I read something on the Airbnb Style Guide (https://github.com/airbnb/javascript/issues/445).